Source/WebCore/page/LocalFrameView.h

```diff
@@ -745,6 +745,9 @@ class LocalFrameView final : public FrameView {
     void updateScrollPositionForScrollAnchoringController() final;
     void invalidateScrollAnchoringElement() final;
 
+    bool usesOverlayScrollbars() const;
+    WEBCORE_EXPORT void overlayScrollbarDefaultDidChange();
```

Comment:
It's a bit odd to see the word "Default" here since that's a term of art for Cocoa, but this is a cross-platform file. Maybe `scrollbarsStyleDidChange`

Reply:
Fixed.